packages/chaire-lib-common/src/config/defaultPreferences.config.ts

```diff
@@ -58,6 +58,10 @@ const defaultPreferences: PreferencesModel = {
                 localizedTitle: 'main:Routing',
                 icon: '/dist/images/icons/interface/routing_white.svg'
             },
+            comparison: {
+                localizedTitle: 'transit:transitComparison:ScenarioComparison',
+                icon: '/dist/images/icons/transit/comparison_white.svg'
+            },
```

Comment:
When you rebase on latest `main`, there will likely be a conflict. Just "accept current change" (the one from main) in defaultPreferences.config.ts and put that block in the `packages/transition-common/src/config/project.config.ts` file instead.